### ISSUE TITLE
rFinder B1

### ISSUE THREAD

**Issue by photochap** [ge0rg/aprsdroid#269]
Hello
is it possible to using rfinder b1's RF module ?
to redirect "radio android" sound to APRSDROID

in your issue : "support for dmr/uhf/vhf radio enabled smartphone (e.g. rfinder) #187", you say "Unfortunately, the Chinese tech is very badly integrated and documented...."

if you can contact Bob Greenberg
 (rfinder conceptor), it's probably possible to some attempts

Thanks you
73

**Comment by KD9OOF** [comment]
I second the request for the RFinder support. Dual directional possibly, although that might be too much. Bob Greenberg can help you in any way and is ready and willing to work with you to make this work.

**Comment by ge0rg** [comment]
Yeah, thanks very much for the reminder. Bob indeed has given me some developer documentation on how to activate the TX mode in the rFinder phone. It's not 100% straight-forward, and I need to set off some time to develop it and to give a beta to Bob. Unfortunately, I don't have the time right now.

Will keep this issue open and post updates.

**Comment by hon1nbo** [comment]
This still on a roadmap somewhere?

**Comment by ge0rg** [comment]
Yes, it's still on the agenda. Unfortunately, the different device generations have different APIs to trigger TX and to route audio, and I'm lacking the docs :(
